src/types/api/jsonrpc/index.ts

```diff
@@ -0,0 +1,29 @@
+// TODO: Provjerit ovo i 'mergat' sa methods
```

Comment:
leftover? or comment in english if not :D

Reply:
leftover